src/man/firejail-profile.txt

```diff
@@ -522,12 +570,6 @@ Disable 3D hardware acceleration.
 Disable automatic ~/.config/pulse init, for complex setups such as remote
 pulse servers or non-standard socket paths.
 .TP
-\fBnodbus
-Disable D-Bus access. Only the regular UNIX socket is handled by
-this command. To disable the abstract socket, you would need to
-request a new network namespace using the net command. Another
-option is to remove unix from protocol set.
-.TP
 \fBnodvd
```

Comment:
`nodbus` would still work and be in the existing profiles, right? If so I'd prefer to leave these lines in as documentation, but mark as deprecated... i.e.
`\fBnodbus (deprecated)`
...

Then once we phase `nodbus` out off **all** profiles, let's remove.


EDIT: or do like the main firejail.txt documentation, line 1280, and specify what nodbus is equivalent to.

Reply:
it was moved: see line 498

Reply:
> Then once we phase nodbus out off all profiles, let's remove.

I would keep it for one release and add a deprecated warning. D-Bus can be used to escape the sandbox in several ways and users might have it in their .locals or custom profiles.

Reply:
I marked the `--nodbus` option and the `nodbus` command as deprecated in the manpages.